stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/reactions/ReactionsViewStyle.kt

```diff
@@ -0,0 +1,28 @@
+package io.getstream.chat.android.ui.messages.reactions
+
+public abstract class ReactionsViewStyle {
```

Comment:
Could we get away with making some (or all) of these new style classes `internal`? I think we only create these in our own code, from attributes, and don't want clients to create instances of them. If they need to use them (hopefully not?), let's at least make their ctors `internal`.

Reply:
Made all the style classes internal